kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -1292,6 +1387,20 @@ def update_shareblock_vm_ha_params(vg_uuids, fencer_cmd=None):
             f.write(jsonobject.dumps(cmd_json))
 
+def stop_vg_heartbeat(vg_uuid):
+    with WRITE_SHAREBLOCKVMHAPARAMS_LOCK:
+        if not os.path.exists(SHAREBLOCK_VM_HA_PARAMS_PATH):
+            return
+        with open(SHAREBLOCK_VM_HA_PARAMS_PATH, 'r+') as f:
+            param = f.read().strip()
+            if len(param) == 0:
+                return
+            param_dict = json.loads(param)
+            if "vgUuids" in param_dict and type(param_dict["vgUuids"]) == list:
```

Ruff (0.12.2)

Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)